e2e/utils.go

```diff
@@ -992,3 +992,39 @@ func pvcDeleteWhenPoolNotFound(pvcPath string, cephfs bool, f *framework.Framewo
 	err = deletePVCAndValidatePV(f.ClientSet, pvc, deployTimeout)
 	return err
 }
+
+func checkMountOptions(pvcPath, appPath string, f *framework.Framework) error {
+	pvc, err := loadPVC(pvcPath)
+	if pvc == nil {
+		return err
+	}
+
+	pvc.Namespace = f.UniqueName
+
+	app, err := loadApp(appPath)
+	if err != nil {
+		return err
+	}
+	app.Labels = map[string]string{"app": "validate-mount-opt"}
+	app.Namespace = f.UniqueName
+
+	err = createPVCAndApp("", f, pvc, app)
+	if err != nil {
+		return err
+	}
+
+	opt := metav1.ListOptions{
+		LabelSelector: "app=validate-mount-opt",
+	}
+
+	mountOpt := "discard"
```

Comment:
I do not like the fact that the `discard` option is hardcoded here. Can it be checked that the option is also part of the PVC request? Or pass the mount option as an argument from the test?

Reply:
made as an argument